Nicklas373/Hana-PDF
# BE: Migrating to Laravel 11.x base

Migrating Hana PDF backend to use latest Laravel 11.x base !

What's changed:
- Enhance JWT Auth logic (Migrate from PHPOpenSourceSaver to Tymon for JWTAuth)
- Enhance all return response on Auth, Core, Data, File and Misc API (Use new simple return response) and also properly return correct error response, instead global error message
- Enhance rate limiting features, now use limit from IP instead of sessions
- Introduce all features and fixes from Laravel 11.x base
- Include application log as new filter on notifyLog, and also include order by for application log
- Implement new exceptions handler for unattended access ! : - Now backend will check if request that came from client, has proper or required request headers and it's actually ajax or xhtml request. Then it'll return as JSON, if not then it'll redirect to uptime page.
	- This exceptions also will check if necessary bearer token are reserved by the client or not, and return redirect to HTML request to prevent API interfaces leaks to browser.
- Migrate and adapt all previous function and features from laravel 10.x base into 11.x
- Properly define HTTP status response for all return response in controller
- Re-define all API routing for request method to correct request method, and also re-mapping for route request to specific route, from their group request
- Update to the latest PHP dependencies
- Use proper function to check HTTP request is success or not (Http::timeout), instead using CURL
- Use proper function to generate UUID (Str::uuid) from laravel, instead of native PHP features
